src/test/app.test.js

```diff
@@ -4,9 +4,9 @@ import _ from 'lodash';
 import fs from 'fs-extra';
 import path from 'path';
 import {assert, test as helpers} from 'yeoman-generator';
-
 import * as libs from './libs';
 import packageJSON from '../generators/app/package-json';
+import {assert as chaiAssert} from 'chai';
```

Comment:
Is there a reason chai is needed here vs using the assert module provided by yeoman? It extends Node's [assert](https://nodejs.org/api/assert.html) module; so, you should have the methods you need available.